src/main/resources/data-repository-openapi.yaml

```diff
@@ -312,11 +421,11 @@ definitions:
       - table
       - column_list
       - cardinality
-      
+
     properties:
       table:
-        $ref: '#/definitions/IdentifierNameModel'
-        
+        $ref: '#/definitions/ObjectName'
+
```

Comment:
below column list should just be column (not an array).

Reply:
You can't assume that a database relationship is single column to single column. There are multi-column primary keys and, therefore, multi-column foreign keys. This is typically done so that you don't have to generate unique ids on every table. I would like to keep the expression in the input as an array.

Contrived example:
patient:
 PK: patient_id

visit:
 PK: (patient_id, visit_timestamp)
 FK: patient_id

measurement:
 PK: (patient_id, visit_timestamp, measurement_type)
 FK: (patient_id, visit_timestamp)

Reply:
@ddietterich When we were done discussing this yesterday I was under the impression that this would be an advanced case that we wouldn't start off supporting. We're not assuming that all relationships we encounter will be one column to one column, but I think at least the first few datasets we deal with will probably satisfy this constraint.